lib/graphql/schema/printer.rb

```diff
@@ -44,6 +48,10 @@ def print_schema_definition(schema)
       BUILTIN_SCALARS = Set.new(["String", "Boolean", "Int", "Float", "ID"])
       private_constant :BUILTIN_SCALARS
 
+      def is_spec_directive(directive)
+        ['skip', 'include', 'deprecated'].include?(directive.name)
+      end
```

Comment:
> non-spec directives 

Part of me wants to leave this out, since there _aren't_ any non-spec directives yet. But ... if we left it out, I can guarantee people would be surprised not to see `@defer` / `@stream` in the query dump if I ever got around to merging that 😬